JBoss/EAP/6/templates/static/oval/jboss_eap_vendor_supported.xml

```diff
@@ -3,8 +3,12 @@
     <metadata>
       <title>JBoss Enterprise Application Platform Supported Version</title>
       <description>Installed version of JBoss is a vendor supported version.</description>
+      <affected family="undefined">
+        <platform>JBoss Enterprise Application Platform 6</platform>
+      </affected>
+      <reference source="CPE" ref_id="cpe:/a:redhat:jboss_enterprise_application_platform:6.0.0" />
```

Comment:
@kharyam please remove the CPE reference. This should be handled by the `<extended_definition />`.

Reply:
@redhatrises removed, thanks